changelogs/unreleased/gh-8169-raft-assert-on-wal-write.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/raft
+
+* Fixed an assertion failure in case a raft candidate is reconfigured to a voter
```

Comment:
Nit: we usually use term 'election' instead of 'raft' in public texts. Many people don't know what 'raft' is.

Reply:
Ok, fixed:
```diff
diff --git a/changelogs/unreleased/gh-8169-raft-assert-on-wal-write.md b/changelogs/unreleased/gh-8169-raft-assert-on-wal-write.md
index 9db961d16..e7f875280 100644
--- a/changelogs/unreleased/gh-8169-raft-assert-on-wal-write.md
+++ b/changelogs/unreleased/gh-8169-raft-assert-on-wal-write.md
@@ -1,4 +1,4 @@
 ## bugfix/raft
 
-* Fixed an assertion failure in case a raft candidate is reconfigured to a voter
-  during an ongoning WAL write (gh-8169).
+* Fixed an assertion failure in case an election candidate is reconfigured to a
+  voter during an ongoning WAL write (gh-8169).
```

I didn't change the heading though, because we always use `bugfix/raft` and never `bugfix/elections`:
```
➜  sptnt git:(raft-wal-write-assert) grep -r '/raft' ./changelogs 
./changelogs/unreleased/promote-hang.md:## bugfix/raft
./changelogs/unreleased/promote-multiple-infinite.md:## bugfix/raft
./changelogs/unreleased/strict_fencing.md:## feature/raft
./changelogs/unreleased/gh-8169-raft-assert-on-wal-write.md:## bugfix/raft
./changelogs/unreleased/gh-7253-split-brain-early-vote.md:## bugfix/raft
./changelogs/unreleased/gh-7253-split-brain-old-term-txn.md:## bugfix/raft                                                      /0,04s
➜  sptnt git:(raft-wal-write-assert) grep -r '/election' ./changelogs                                                           /0,03s
➜  sptnt git:(raft-wal-write-assert) 
```